src/rendering/rendering.zig

```diff
@@ -44,8 +44,6 @@ pub const ContextType = enum {
     pub fn fromData(comptime Data: type) ContextType {
         if (Data == json.Value or (trait.isSingleItemPtr(Data) and meta.Child(Data) == json.Value)) {
             return .json;
-        } else if (Data == json.ValueTree or (trait.isSingleItemPtr(Data) and meta.Child(Data) == json.ValueTree)) {
-            return .json;
```

Comment:
I'm not completely sure about this bit. It looks like json.Value is now the only way to do dynamic json and ValueTree was deprecated.